server/views/applications/pages/risk-to-self/current-risk.njk

```diff
@@ -12,6 +12,39 @@
     {{dateOfOasysImport(date)}}
   {% endif %}
 
+  {% set currentRiskDetailHintHtml %}
+  <p class="govuk-hint">If known, provide details including:</p>
+
+  <ul class="govuk-list govuk-list--bullet govuk-hint">
+    <li>
+        if they are assessed at imminent risk of self harm or suicide and require ongoing monitoring to reduce the risk
```

Comment:
pedant corner but does this work better as "if" or "whether"?

Reply:
good shout, have made the change and squashed/squished/squelched